test/fail_compilation/fail19757.d

```diff
@@ -0,0 +1,8 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19757.d(8): Error: cannot implicitly convert expression `"oops"` of type `string` to `ulong`
```

Comment:
The error message is `uint` on 32 bit platforms.

Reply:
Is there any way I can version the error?

Reply:
cc @ibuclaw you did some test parameterisation recently.